src/app/Plans/components/PlanStatus.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { IMigration, IPlan } from '@app/queries/types';
+import { StatusIcon, StatusType } from '@konveyor/lib-ui';
+import { Level, LevelItem, Progress, ProgressMeasureLocation } from '@patternfly/react-core';
+import spacing from '@patternfly/react-styles/css/utilities/Spacing/spacing';
+
+interface IPlanStatusProps {
+  plan: IPlan;
+  migration: IMigration;
+}
+
+const PlanStatus: React.FunctionComponent<IPlanStatusProps> = ({
+  plan,
+  migration,
+}: IPlanStatusProps) => {
+  if (plan.status.conditions.every((condition) => condition.type === 'Ready')) {
+    return <StatusIcon status={StatusType.Ok} label="Ready" />;
+  } else {
+    // TODO: This is only a placeholder, more statuses check needs to be done.
+    const totalVMs = plan.spec.vmList.length;
+    const percentVMsDone = totalVMs > 0 ? (migration.status.nbVMsDone * 100) / totalVMs : 0;
+    const label = 'Running';
+
+    return (
+      <>
+        <Level>
+          <LevelItem>
+            <StatusIcon status={StatusType.Warning} label={label} />
+          </LevelItem>
+          <LevelItem className={`${spacing.mbXs} ${spacing.prMd}`}>
```

Comment:
Sorry to keep nitpicking @gildub but can you put these classes on the whole `<Level>` instead of just the `<LevelItem>`? the first item isn't vertically aligned anymore with this

Reply:
No worries, done.
It makes sense but I don't see much of a difference. Maybe I'm missing something.

Reply:
It's subtle, but it's there. The right side was a few pixels higher. I know, nitpicky 😄 
<img width="308" alt="Screen Shot 2020-09-24 at 10 42 45 AM" src="https://user-images.githubusercontent.com/811963/94161005-3b5e5500-fe53-11ea-8e00-3e94ff564c4b.png">

<img width="346" alt="Screen Shot 2020-09-24 at 10 42 28 AM" src="https://user-images.githubusercontent.com/811963/94161015-3d281880-fe53-11ea-99d9-9e2ad4d42157.png">

Reply:
Oh I see, reducing my screen helps!
Well in UX/UI every pixel matter!
Thanks